athf/commands/mcp.py

```
@@ -0,0 +1,34 @@
+      {
+        "athf": {
+          "command": "athf-mcp",
+          "args": ["--workspace", "/path/to/hunts"]
+        }
+      }
+    """
+    try:
+        from athf.mcp.server import main as mcp_main
+    except ImportError:
+        click.echo("Error: MCP dependencies not installed. Install with: pip install 'athf[mcp]'", err=True)
+        raise SystemExit(1)
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)